guides/assets/stylesheets/main.css

```diff
@@ -238,6 +238,26 @@ body {
   }
 }
 
+.clipboard-button {
+  background: #dbdbdb;
+  border: 0;
+  border-radius: 5px;
+  padding: 2px 10px;
+  position: absolute;
+  bottom: 2px;
+  right: 2px
+}
+
+.clipboard-button:active {
+  background: #ccc;
+}
+
+.clipboard-content {
+  opacity:0;
+  position:absolute;
+  z-index:-1;
+}
```

Comment:
Looks like this is missing from `main.rtl.css`.

Reply:
Missed that. Thanks! 😄